libs/accounts/src/lib/breakdown-table.tsx

```diff
@@ -5,19 +5,47 @@ import {
   PriceCell,
   t,
 } from '@vegaprotocol/react-helpers';
-import type { VegaValueFormatterParams } from '@vegaprotocol/ui-toolkit';
 import {
   AgGridDynamic as AgGrid,
+  Intent,
   progressBarCellRendererSelector,
 } from '@vegaprotocol/ui-toolkit';
 import { AgGridColumn } from 'ag-grid-react';
 import type { AgGridReact, AgGridReactProps } from 'ag-grid-react';
 import type { AccountFields } from './accounts-data-provider';
 import { AccountTypeMapping } from '@vegaprotocol/types';
-import {
-  progressBarHeaderComponentParams,
-  progressBarValueFormatter,
-} from './accounts-table';
+import type {
+  ValueProps,
+  VegaValueFormatterParams,
+} from '@vegaprotocol/ui-toolkit';
+import type { ValueFormatterParams } from 'ag-grid-community';
+
+export const progressBarValueFormatter = ({
+  data,
+  node,
+}: ValueFormatterParams): ValueProps['valueFormatted'] | undefined => {
+  if (!data || node?.rowPinned) {
+    return undefined;
+  }
+  const min = BigInt(data.used);
+  const mid = BigInt(data.available);
+  const max = BigInt(data.deposited);
+  const range = max > min ? max : min;
```

Comment:
comment (non-blocking): usually i find using `Math.max(min, max)` easier to use for cases like this, because i don't have to think the ternary branching. but this is absolutely fine as it is.